src/Error/ExceptionRenderer.php

```diff
@@ -365,6 +366,19 @@ protected function _template(Throwable $exception, string $method, int $code): s
         return $this->template = $method;
     }
 
+    /**
+     * Gets fallback template for rendering exception info.
+     *
+     * @param \Throwable $exception Exception instance.
+     * @param string $method Method name.
+     * @param int $code Error code.
+     * @return string Template name
+     */
+    protected function _fallbackTemplate(Throwable $exception, string $method, int $code): string
```

Comment:
This could also be `_missingTemplate` to be more specific about when it is used.